test/specs/xsrf.spec.js

```diff
@@ -28,6 +28,19 @@ describe('xsrf', function () {
     });
   });
 
+  it('should not set xsrf header if xsrfCookieName is null', function (done) {
```

Comment:
Maybe you should include a test to check that `document.cookie` isn't read if `xsrfCookieName` is null, as it's the main motivation of this PR. We can only do that using ES5 getters and we haven't used them in any test (nor the production code). It will work in the test runners but we claim to support IE8+ for the production code, so I'm not entirely sure it's a good idea. What do you think @mzabriskie @nickuraltsev ?